sonar-plugin-api/src/main/java/org/sonar/api/measures/MeasureUtils.java

```diff
@@ -19,17 +19,17 @@
  */
 package org.sonar.api.measures;
 
-import org.apache.commons.lang.StringUtils;
-
-import javax.annotation.Nullable;
-
 import java.util.Collection;
+import javax.annotation.Nullable;
+import org.apache.commons.lang.StringUtils;
 
 /**
  * An utility class to manipulate measures
  *
  * @since 1.10
+ * @deprecated since 5.6. {@link Measure} is deprecated.
  */
+@Deprecated
 public final class MeasureUtils {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)